helm-charts/sawmills-collector/tests/sibling_fallback_test.yaml

```
@@ -440,12 +440,61 @@ tests:
     set:
       haproxy.sibling_fallback.enabled: true
+      haproxy.external_fallback.enabled: true
+      haproxy.mapping:
+        http_4318:
+          from: 4318
+          to:
+            port: 4318
+    asserts:
+      - matchRegex:
+          path: data["haproxy.cfg"]
+          pattern: "acl is_sibling_hop hdr\\(X-Sibling-Hop\\) -m found"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)